src/core/lv_obj_style.c

```diff
@@ -224,7 +224,7 @@ void lv_obj_enable_style_refresh(bool en)
 
 lv_style_value_t lv_obj_get_style_prop(const lv_obj_t * obj, lv_part_t part, lv_style_prop_t prop)
 {
-    lv_style_value_t value_act;
+    lv_style_value_t value_act = {.num = 0, .ptr = NULL, .color.full = 0};
```

Comment:
It produces a warning. Maybe `lv_style_value_t value_act = { 0 }`

Reply:
Hi @kisvegabor, I have pushed a new commit. I realised that `value_act` is a union and for some reason I was thinking it is a struct. I have now just initialised the `num` member to zero 😊